route/build/src/remap.f90

```diff
@@ -260,49 +260,69 @@ subroutine remap_1D_runoff(runoff_data_in, remap_data_in, basinRunoff, ierr, mes
   end subroutine remap_1D_runoff
 
   ! *****
-  ! public subroutine: assign runoff data in runoff layer to hru in river network layer
+  ! public subroutine: assign (sort) fluxes and states from a given ID array to another ID array
   ! ***************************************************************************************************************
-  ! case 3: hru in runoff layer is hru polygon identical to river network layer (stored in 1-dimension array)
-  subroutine sort_runoff(runoff_data_in, basinRunoff, ierr, message)
+  ! case 3: sort a given input flux and its ID based on a second given ID array,
+  ! hru in runoff layer is hru polygon identical to river network layer (stored in 1-dimension array)
+  subroutine sort_flux  (ID_in,            & ! input: the array of ids for HRU/seg (can be from 1 to nHRU or nSeg)
```

Comment:
I would remove the part `(can be from 1 to nHRU or nSeg)` since id is not necessarily sequential number.